config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  resources :package_sets, except: [:new, :edit]
```

Comment:
this is outside the `/api/v1/` scope. is this intended?

Reply:
oh, I used `rails generate scaffold` that's why. Will fix it